hedera-node/test-clients/src/main/java/com/hedera/services/bdd/suites/regression/UtilScalePricingCheck.java

```diff
@@ -54,7 +53,7 @@ public List<HapiSpec> getSpecsInSuite() {
         return List.of(nftPriceScalesWithUtilization());
     }
 
-    @HapiTest
+    // Can only be run against an isolated network since it assumes the state begins with 0 NFTs minted
```

Comment:
Does anything need to be done about this, or do we leave it as is?

Reply:
We could consider adding an `int order() default 0` (lower number is higher precedence) to the `@HapiTest` annotation and then mark this `@HapiTest(order=Integer.MIN_VALUE)`.

Reply:
Hmm, yeah something to think about